pipelines/wan2_1/blocks/clean_kv_cache.py

```diff
@@ -114,6 +114,15 @@ def __call__(self, components, state: PipelineState) -> tuple[Any, PipelineState
 
         # Run the generator with the clean latent at timestep = 0 to update the KV cache
         conditional_dict = {"prompt_embeds": block_state.conditioning_embeds}
+        context_timestep = (
```

Comment:
L100-110 can be removed given this, but let's preserve the comments above those lines here.